### ISSUE TITLE
Cannot include file from the same folder

### ISSUE THREAD

**Issue by Misiur** [pawn-lang/compiler#504]
Environment: Windows 10 -> WSL ubuntu (optional -> docker ubuntu)

https://github.com/Misiur/compiler-poc

Fails to compile due to not finding the correct include from relative path

**Comment by Y-Less** [comment]
Try `inc/poci`.  Unfortunately include paths are horrible in pawn.

**Comment by Misiur** [comment]
I'm fully aware of historical intricaties of include paths. Does that mean I can finally use `YSI/y_foreach` and such?

**Comment by Y-Less** [comment]
I've tried very hard to make YSI work with both `/` and `\` - basically by just internally trying about 6 different variations of possible include path.  However, it isn't perfect.  The compiler generally uses the OS native separator, so `/` on *nix and `\` on Windows.  WSL I'm guessing may use `\` not `/`, while most people use Windows hence the instruction to use `\` for them.

**Comment by Misiur** [comment]
YSI does not work with `/` on linux:
`/app/pawno/include/YSI/y_iterate.inc(76) : fatal error 100: cannot read from file: "..\YSI_Data\y_iterate"`

**Comment by Misiur** [comment]
But it also fails with `\` when included from `/`:
`ugrp/includes.inc(16) : fatal error 100: cannot read from file: "YSI\y_iterate"`

**Comment by AmyrAhmady** [comment]
how are you not getting any user defined error while trying to include from `YSI` directory? are you on 4.x? cause it's kinda different in 5.x

**Comment by Y-Less** [comment]
Yes.  I've documented this weird behaviour somewhat extensively, but it is a mess.  However, I was just talking about your original problem in your PoC, not YSI (because you offered $5).

Also what @AmyrAhmady said.
